src/Particle/DistanceTableData.h

```diff
@@ -398,6 +398,15 @@ struct DistanceTableData
       Temp.resize(N[SourceIndex]);
     }
   }
+
+  /// Description generated during creation, to be printed later
+  std::ostringstream description;
```

Comment:
Hopefully in the near future, we will change most objects of DistanceTableData to private.
I'm thinking of keeping the class light.
On the other hand, this printout doesn't need to be kept the whole life time of DistanceTableData.
For this reason, is it better if we expand the createDistanceTable by introducing 1st argument a stream and print to the stream. The stream is stored in ParticleSet.